src/common/constants/api.js

```diff
@@ -3,6 +3,7 @@ export const BATCH_GROUPS_API = 'batch-groups';
 export const BATCH_VOUCHERS_API = 'batch-voucher/batch-vouchers';
 export const BATCH_VOUCHER_EXPORTS_API = 'batch-voucher/batch-voucher-exports';
 export const CONFIG_API = 'configurations/entries';
+export const EXCHANGE_RATE_API = 'finance/exchange-rate';
```

Comment:
please check if new interface and permissions should be added to package.json

Reply:
👌